tarodb/sqlite/queries/assets.sql

```diff
@@ -19,7 +19,7 @@ FROM asset_minting_batches batches
 JOIN internal_keys keys
     ON batches.batch_id = keys.key_id
 LEFT JOIN chain_txns txs
-    ON batches.genesis_id = txs.txn_id
+    ON batches.genesis_id in (SELECT genesis_id FROM genesis_points WHERE anchor_tx_id = txs.txn_id)
```

Comment:
See comment on prior commit, basically we can just drop this entirely then rely on the height hint in the transaction inserted into the batch later on.

Reply:
Yep 👍